README.md

```diff
@@ -13,7 +13,9 @@ In order to authenticate as a GitHub App, you need to generate a Private Key
 and use it to sign a JSON Web Token (jwt) and encode it. 
 
 ```js
-const auth = await appAuth({id, privateKey})
+const app = require('@octokit/app.js')
+
+const auth = await app.appAuth({id, privateKey})
```

Comment:
That’s interesting. I intuitively thought that having an `OctokitApp` constructor with an `app` instance would be a better fit that having a singleton `app` module with stateless methods on it. Could you elaborate on your thoughts here?

For comparison, I thought of something like this

```js
const App = require('@octokit/app.js')
const app = new App({id, privateKey})
const appAuth = await app.appAuth()
```

Reply:
That makes perfect sense! I've never really done something like this before, but now that you explain it I think your way is better :)